src/pages/Dashboard/RecentTransactions.tsx

```diff
@@ -27,31 +26,22 @@ type RecentTransactionProps = {
 
 type RecentTransactionsBodyProps = {
   limit: number
-} & (
-  | {
-      txIds: TxId[]
-      filter?: never
-    }
-  | {
-      txIds?: never
-      filter: RecentTransactionFilter
-    }
-)
+} & {
+  filter: RecentTransactionFilter
+}
 
 const defaultFilter = {}
 
 export const RecentTransactionsBody: React.FC<RecentTransactionsBodyProps> = ({
-  txIds: _txIds,
```

Comment:
Unused - no reason to pass a bunch of `txIds` down to `<RecentTransactionsBody />` from `<RecentTransactions />` unless we need to